tests/nested/manual/hybrid-remodel/task.yaml

```diff
@@ -16,7 +16,10 @@ environment:
 
 prepare: |
   VERSION="$(tests.nested show version)"
-  snap download --basename=pc-kernel --channel="$VERSION/stable" pc-kernel
+
+  # download beta to increase the unlikeliness that we encounter the
+  # same revision when remodelling. 
```

Comment:
So this can still randomly fail. What kind of option do we have for this to be fully robust regardless of releases?

Reply:
Idk, I was hoping someone might have a better idea - maybe repacking it and installing as an unasserted kernel would be fine

Reply:
Why do not we repack the kernel here?

Reply:
Note that we will have to rewrite this test in the FDE branch. Because we will have to repack everything and remodel from the fakestore. So maybe we do not need to do the effort twice.